dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -4839,7 +4851,7 @@ private AssignmentNode createIncrementOrDecrementAssign(
 
             if (target == null) {
                 target = treeBuilder.buildAssignment(exprTree, (ExpressionTree) narrowed.getTree());
-                handleArtificialTree(target);
+                handleArtificialTree(target, TreePath.getPath(getCurrentPath(), exprTree));
```

Comment:
add document in dataflow-manual, to demonstrate how this works for postfix operation.

Reply:
https://github.com/opprop/checker-framework/blob/master/dataflow/manual/content.tex#L979
seems like the right area for some extra explanation.